action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
     description: Name of the branch where the sphinx documentation is located
     required: false
     default: 'main'
+  branch-checkout-args:
+    description: Arguments to pass to `git checkout`: `git checkout ${checkout-args} "${branch}"`
+    required: false
+    default: ''
```

Comment:
https://git-scm.com/docs/git-checkout
https://git-scm.com/docs/git-clone

Reply:
@westurner, sorry for my super late reply. I am going back to the maintenance of this repository.

I have to have a look at your changes. Do you still think that this improves the action? I didn't understand if your "This is maybe not necessary" affects your whole PR. 

Again, sorry. I hope you still want to discuss about it.

Reply:
Thanks for coming back to this project! Before you merge, please also add quotes around the description value since it contains a colon which the YAML parser does not expect.